src/Helpers/ProjectInfo.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EcEuropa\Toolkit\Helpers;
+
+use EcEuropa\Toolkit\TaskRunner\AbstractCommands;
+use Robo\Contract\VerbosityThresholdInterface;
+
+class ProjectInfo extends AbstractCommands
+{
+
+    /**
+     * Helper function to get enabled Drupal modules.
+     */
+    public function getEnabledDrupalModules() {
+        $enabledModules = [];
+        $result = $this->taskExec($this->getBin('drush') . ' pm-list --fields=status --format=json')
```

Comment:
What do you think of using the `pm:list` command option `status`, the command could be:
```
drush pm-list --status=enabled --format=json
```

https://www.drush.org/12.x/commands/pm_list/

Reply:
Hi @joaocsilva this has been fixed, please proceed with the review.